bindata/v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrole.yaml

```diff
@@ -15,7 +15,8 @@ rules:
 - apiGroups:
   - networking.k8s.io
   resources:
-  - ingress
+  - ingresses
+  - ingressclasses
```

Comment:
This looks off to me.  Can you create another cluusterrole that is used to establish watches and grant write event permissions?

The permissions for this particular controller should only be those permissions needed for the control loop, which are usually the write permissions.

Reply:
we are already giving these persmissions via shared informers and `openshift-controller-manager-sa` https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/bindata/v3.11.0/openshift-controller-manager/informer-clusterrole.yaml

I am doing something similar in the split https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/d1ad600a0c86a3b808e529bb622a7b2a3e33b17e/bindata/v3.11.0/openshift-controller-manager/route-controller-informer-clusterrole.yaml

this change here was only for declaring the RBAC dependency of the `ingress-to-route-controller`